### PR TITLE
Chagne spec file to be from pyp2rpm

### PR DESCRIPTION
**Pull request by djw8605**

### CODE REVIEW

File: configs/python-scitokens.spec [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ln -s %{_bindir}/scitokens-admin-create-key-%{python2_version} %{buildroot}/%{_bindir}/scitokens-admin-create-key-2
+cp %{buildroot}/%{_bindir}/scitokens-admin-create-token %{buildroot}/%{_bindir}/scitokens-admin-create-token-%{python2_version}
+ln -s %{_bindir}/scitokens-admin-create-token-%{python2_version} %{buildroot}/%{_bindir}/scitokens-admin-create-token-2
+
+
+%files -n python2-%{pypi_name}
+%doc README.rst
+%{_bindir}/scitokens-admin-create-key
+%{_bindir}/scitokens-admin-create-key-2
+%{_bindir}/scitokens-admin-create-key-%{python2_version}
```
Comment (bbockelm) [inline configs/python-scitokens.spec]: I'm not clear on what this does.  Why are there three versions of this binary in the end?

File: configs/python-scitokens.spec [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+%package -n     python2-%{pypi_name}
+Summary:        %{summary}
+Provides:       python-%{pypi_name}
+ 
+Requires:       python2-jwt
+Requires:       python2-cryptography
+%description -n python2-%{pypi_name}
+
+
```
Comment (bbockelm) [inline configs/python-scitokens.spec]: Clear out excessive whitespace here.

**Comment by djw8605** [comment]
This spec file was prepared by `pyp2rpm` tool.  But the python3 stuff didn't work on el7 builds, so I removed it.  There are undefined macros used, specifically the `python_provide` macro.

**Comment by bbockelm** [comment]
Alright, if you open an issue for the Python 3 packaging, I'll ACK this one.

**Comment by djw8605** [comment]
Opened issue #46 for python3 RPM packaging.

**Pull request merged**